qutebrowser/misc/guiprocess.py

```diff
@@ -179,3 +184,9 @@ def start_detached(self, cmd, args):
 
     def exit_status(self):
         return self._proc.exitStatus()
+
+    def final_stdout(self):
+        return self._final_stdout
+
+    def final_stderr(self):
+        return self._final_stderr
```

Comment:
I don't see a need for getter functions here - instead, store them as public attributes (`self.final_stdout` and `final_stderr`) and access them directly.

Reply:
Fixed